salt/modules/mod_random.py

```diff
@@ -122,3 +123,21 @@ def shadow_hash(crypt_salt=None, password=None, algorithm='sha512'):
         salt '*' random.shadow_hash 'My5alT' 'MyP@asswd' md5
     '''
     return salt.utils.pycrypto.gen_hash(crypt_salt, password, algorithm)
+
+def rand_int(start=1, end=10):
+    '''
+    .. versionadded: 2015.5.1
```

Comment:
@jfindlay yes you are right, and if the code looks good please merge it in 2015.5
I have change the version